src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -93,6 +93,13 @@
     <!--     
       Exclude dependencies that are not inserted from Roslyn repo.
       Note: can't use globs due to https://github.com/Microsoft/msbuild/issues/3791.
+      
+      these dependency should be removed once StreamJsonRpc adds a build target for
+      net472 - https://github.com/dotnet/roslyn/issues/34558
+      
+      <_Dependency Remove="Nerdbank.Streams"/>
+      <_Dependency Remove="System.IO.Pipelines"/>
+      <_Dependency Remove="System.ValueTuple"/>
```

Comment:
As @sharwell mentioned here [https://github.com/dotnet/roslyn/issues/34558](https://github.com/dotnet/roslyn/issues/34558), should we consume 2.1 and remove them?

Reply:
This is the only concern I have. Other than that, the PR looks good.

---
In reply to: [277095521](https://github.com/dotnet/roslyn/pull/34860#discussion_r277095521) [](ancestors = 277095521)

Reply:
StreamJsonRpc 2.1 isn't stable yet. But 2.0.167 has what you're looking for.

Reply:
Thank you, @AArnott ! If 2.0.167 does not have those extra references, let us move forward to 2.0.167 and remove them.

Reply:
2.0.167 includes a net472 target and thus drops the System.ValueTuple reference. Nerdbank.Streams and System.IO.Pipelines references remain.

Reply:
sure. I can move to newer version.